packages/data-clients/core/src/request-utils.test.ts

```diff
@@ -0,0 +1,16 @@
+import { jsonBody } from './request-utils';
```

Comment:
I am not convinced of the necessity of this test, it mostly checks the behavior of a library.

Reply:
yeah, I didn't initially include it, but miguel requested it in a dm for thoroughness and I figured it couldn't hurt